TC_project/TwinCAT Project1/TwinCAT Project1/Untitled1/DUTs/DUT_STRUCT_TEST.TcDUT

```diff
@@ -1,30 +1,27 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4020.12">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
   <DUT Name="DUT_STRUCT_TEST" Id="{64b6492a-8751-410c-8dcd-b0346af5655e}">
     <Declaration><![CDATA[TYPE DUT_STRUCT_TEST :
 STRUCT
 
 	{attribute 'pytmc' := '
-		pv: ADS_TST:SetfAmpRB
-		pv: ADS_TST:SetfAmp
+		pv: ADS_TST:fAmp
 		io: o
-		pv: ADS_TST:GetfAmp
+		pv: ADS_TST:ReadOnly:fAmp

```

Comment:
I thought we decided to stick with the `_RBV` formatting? Won't pytmc automatically add these now?

Reply:
Yes, you're right. Though this repo is labeled 'example', I think our intention was to use it more as a pytmc test project.

So: you'll notice that the above PV `ADS_TST:fAmp`, marked `io: o`, will create both `ADS_TST:fAmp` and `ADS_TST:fAmp_RBV`. The `:ReadOnly:` portion (which will generate only `ADS_TST:ReadOnly:fAmp_RBV`) will just make it easier for us to differentiate the generated `io: o` vs `io: i` PVs from the IOC.